client/document/desktop.jsx

```diff
@@ -91,8 +91,9 @@ class Desktop extends React.Component {
 							<a
 								className="bug-report"
 								href={ feedbackURL }
-								title="Report an issue"
+								rel="noopener"
```

Comment:
probably want `noopener noreferrer` as per https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
alternatively you could use `components/external-link` which also has the protections

Reply:
(for ref, `noopener` was added to firefox 52, we're currently on 59)

Reply:
> probably want `noopener noreferrer` as per https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Uh right, not sure why I keep forgetting `noreferrer`

> alternatively you could use `components/external-link` which also has the protections

Good idea, will do :+1: